src/test/java/org/entando/plugins/pda/core/service/task/FakeTaskFormService.java

```diff
@@ -0,0 +1,75 @@
+package org.entando.plugins.pda.core.service.task;
+
+import static org.entando.plugins.pda.core.utils.TestUtils.TASK_FORM_PROP_1;
+import static org.entando.plugins.pda.core.utils.TestUtils.TASK_FORM_PROP_2;
+import static org.entando.plugins.pda.core.utils.TestUtils.TASK_FORM_PROP_DESCRIPTION_1;
+import static org.entando.plugins.pda.core.utils.TestUtils.TASK_FORM_PROP_DESCRIPTION_2;
+import static org.entando.plugins.pda.core.utils.TestUtils.TASK_FORM_PROP_KEY_1;
+import static org.entando.plugins.pda.core.utils.TestUtils.TASK_FORM_PROP_KEY_2;
+import static org.entando.plugins.pda.core.utils.TestUtils.TASK_ID_1;
+import static org.entando.plugins.pda.core.utils.TestUtils.TASK_ID_2;
+import static org.entando.plugins.pda.core.utils.TestUtils.TASK_NAME_1;
+import static org.entando.plugins.pda.core.utils.TestUtils.TASK_NAME_2;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.entando.plugins.pda.core.engine.Connection;
+import org.entando.plugins.pda.core.exception.TaskNotFoundException;
+import org.entando.plugins.pda.core.model.form.Form;
+import org.entando.plugins.pda.core.model.form.FormField;
+import org.entando.plugins.pda.core.model.form.FormFieldInteger;
+import org.entando.plugins.pda.core.model.form.FormFieldText;
+import org.entando.plugins.pda.core.model.form.FormFieldType;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FakeTaskFormService implements TaskFormService {
+
+    public static final List<Form> TASK_FORMS = new ArrayList<>();
+
+    static {
```

Comment:
should have at least one boolean field for coverage purposes

Reply:
Done